dev-docs/bidders/freedomadnetwork.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: bidder
+title: Freedom Ad Network
+description: Freedom Ad Network Bidder Adapter
+biddercode: freedomadnetwork
+aliasCode: fan
+gvl_id: none
+coppa_supported: true
+media_types: banner, native
+multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
+safeframes_ok: false
+deals_supported: false
+floors_supported: false
+fpd_supported: false
+pbjs: true
+prebid_member: false
+sidebarType: 1
+tcfeu_supported: false
+dsa_supported: false
+usp_supported: false
+gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
```

Comment:
How do you support tcfeu if you have no gvlid?

Reply:
I didn't know if I could leave the list empty, or omit the key entirely, or list all the defaults and use false in the `tcfeu_supported` key as a definition of not having it.

Reply:
I have removed the key. Is that OK? Or should it be `[]`?